packages/form/addon/lib/field.js

```diff
@@ -413,6 +413,7 @@ export default class Field extends Base {
    * - `form`: Legacy property pointing to the root form.
    * - `info.form`: The form this question is attached to.
    * - `info.formMeta`: The meta of the form this question is attached to.
+   * - `info.mainCaseForm`: The main cases' form (can give useful context in task forms).
```

Comment:
Might make sense to extend this a little bit:

- `info.case.form`
- `info.case.workflow`
- `info.root.case.form`
- `info.root.case.workflow`

What do you think?

Reply:
Me likey!

Reply:
👍 good idea!

Reply:
After implementing the backend part, I think it should be `case.root` instead of `root.case`, because `root` refers to the currently open "root document" (task form), not to the root case document (case form):
- `info.form` => `document.form.slug`
- `info.root.form` => `document.family.form.slug`
- `info.case` => `document.work_item.case`
- `info.case.root` => `document.work_item.case.family`
- `info.case.root.form` => `document.work_item.case.family.form.slug`

Reply:
@anehx added a commit that also makes this feature work with case forms (instead of just task forms) :+1: